scripts/workflow/deploy-service.sh

```diff
@@ -80,7 +95,15 @@ echo "Replacing previous version: ${PREVIOUS_VERSION}"
 echo "Tagging time of deployment to ${ENVIRONMENT} of artefact ${ARTEFACT_BUCKET_NAME}/${WORKSPACE}/${COMMIT_HASH}/${DEPLOYMENT_FILE_NAME}"
 
 DEPLOYED_AT=$(date '+%Y-%m-%d %H:%M:%S')
+
```

Comment:
do the lines above (91 and 95) need the if custom artefact check too ?

Reply:
To be added